config/default-disco.js

```diff
@@ -51,6 +52,12 @@ module.exports = {
       mediaSrc: [staticHost],
     },
   },
+  taarParamsToUse: [
+    'branch',
+    'platform',
+    'study',
+    'telemetry-client-id',
```

Comment:
Might be worth a comment here too to say that this is translated to this from clientId.

Reply:
If you move the mapping to where the filtering happens this config could specify `clientId` instead, which might be easier to understand.

I'm not sure about the merging of the query params with the bits extracted from the url string e.g. `branch` is a query param wheras `platform` comes from the URL, it might be easier to understand if those are kept separate.